server/src/test/resources/configs/row-hmac-check-disabled.yaml

```diff
@@ -0,0 +1,157 @@
+server:
+  applicationConnectors:
+    - type: resources-https
+      port: 4445
+      keyStorePath: dev_and_test_keystore.p12
```

Comment:
Unfortunately (depending on how you look at it), the large number of required values in our configuration file means that testing serialization of a single property requires a large amount of boilerplate.